Patches/Alpha Animals/ThingDefs_Misc/Alpha_Hediffs.xml

```diff
@@ -1,28 +1,49 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Patch>
-  <Operation Class="PatchOperationSequence">
-	<operations>
-	  <li Class="PatchOperationFindMod">
-			
-		<mods><li>Alpha Animals</li></mods>
-			
-		<match Class="PatchOperationSequence">
+	<Operation Class="PatchOperationSequence">
 		<operations>
-
-
-            <li Class="PatchOperationReplace">
-				<xpath>Defs/HediffDef[defName="AA_BurrowingUnderSand"]/stages/li/statOffsets</xpath>
-				<value>
-                                                                           <statOffsets>
-					<ArmorRating_Sharp>3</ArmorRating_Sharp>
-					<ArmorRating_Blunt>4.5</ArmorRating_Blunt>
-                  			           </statOffsets>
-				</value>
+			<li Class="PatchOperationFindMod">
+				<mods>
+					<li>Alpha Animals</li>
+				</mods>
+				<match Class="PatchOperationSequence">
+					<operations>
+						<li Class="PatchOperationReplace">
+							<xpath>Defs/HediffDef[defName="AA_BurrowingUnderSand"]/stages/li/statOffsets</xpath>
+							<value>
+								<statOffsets>
+									<ArmorRating_Sharp>3</ArmorRating_Sharp>
+									<ArmorRating_Blunt>4.5</ArmorRating_Blunt>
+								</statOffsets>
+							</value>
+						</li>
+						<li Class="PatchOperationReplace">
+							<xpath>Defs/HediffDef[defName="AA_AnoleGrown"]/stages/li/statOffsets</xpath>
+							<value>
+								<statOffsets>
+									<ArmorRating_Sharp>3</ArmorRating_Sharp>
+									<ArmorRating_Blunt>4.5</ArmorRating_Blunt>
+								</statOffsets>
+							</value>
+						</li>
+						<li Class="PatchOperationReplace">
+							<xpath>Defs/HediffDef[defName="AA_AnoleGrown"]/comps/li[Class="HediffCompProperties_VerbGiver"]/tools/</xpath>
+							<value>
+								<li Class="CombatExtended.ToolCE">
+								<label>claws</label>
+								<capacities>
+									<li>Scratch</li>
+								</capacities>
+								<power>32</power>
+								<cooldownTime>2</cooldownTime>
+								<soundMeleeHit>MeleeHit_BionicPunch</soundMeleeHit>
+								<soundMeleeMiss>MeleeMiss_BionicPunch</soundMeleeMiss>
+								<armorPenetrationSharp>2</armorPenetrationSharp>
+								<armorPenetrationBlunt>3.5</armorPenetrationBlunt>
```

Comment:
That's not how CE works AFAIK. You need to specify attack power and penetration for each usable appendage. Also, vanilla attack set is pretty lackluster for a giant intelligent lizard. I used an adult atispec as basis (a very strong creature), and in vanilla version of the mod a grown anole is even better in terms of offensive capabilities (if you only count the claws in).

Reply:
The Hediff you did for the anole resulted in an error for the patch. I could do the success part for the sequence but idk if that will work.